lib/evse_security/evse_security.cpp

```diff
@@ -152,9 +154,13 @@ EvseSecurity::~EvseSecurity() {
 InstallCertificateResult EvseSecurity::install_ca_certificate(const std::string& certificate,
                                                               CaCertificateType certificate_type) {
     EVLOG_debug << "Installing ca certificate: " << conversions::ca_certificate_type_to_string(certificate_type);
+
+    // Some CSMS may send malformed PEM strings. Canonicalize them here.
```

Comment:
nit: I would not refer `CSMS` here; could be used by any consumer of the lib

```suggestion
    // Function may be called with malformed PEM strings. Canonicalize them here.
```